example/functions/getObservedResourceCondition/README.md

```diff
@@ -0,0 +1,39 @@
+# getObservedResourceCondition
+
+## Usage
+
+`getObservedResourceCondition $request $resourceName $conditionType`
+
+Examples:
+
+```golang
+// Print whole condition
+{{ getObservedResourceCondition . "project" "Ready" | toYaml }}
```

Comment:
just to avoid having to deserialize the whole request, would it make sense to have it as a function to lookup a specific condition from a resource:
```
{{ getResourceCondition .observed.resources["project"] "Ready" | toYaml }}
```
or maybe even from a list of conditions:
```
{{ getCondition .observed.resources["project"].status.conditions "Ready" | toYaml }}
```

sure this would mean having to have an if to check the resource is there before calling it, but on the other hand it would be pretty straight forward and also support getting conditions in the composite resource out of the box.

Reply:
Yeah, I understand where you coming from. I thought of this function primarily as a syntactic sugar to cleanup the go templates but I think it makes sense to just give a resource as parameter, and give the responsibility to the user to ensure that input is actually a resource and not null/whatever.